site/docs/components/tag/examples.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title:
+  $ref: ./#/title
+layout: DetailComponent
+sidebar:
+  exclude: true
+data:
+  $ref: ./#/data
+---
+
+<LivePreviewControls>
+<LivePreview componentName="tag" exampleName="Primary">
+
+## Primary
+
+Primary tags feature subtle background colors by default, making them ideal for presenting information or content without being overly attention-grabbing.
```

Comment:
"without be overly attention-grabbing" I'm not a huge fan of this phrasing can we update it please.

Reply:
We can address the content feedback later.